notebooks/1_table_oriented.ipynb

```diff
@@ -0,0 +1,484 @@
+{
```

Comment:
<p>I've only ever seen the word "latter" used when there are two things discussed ("former" and "latter"). Since there are three things being discussed in this case, I would replace "the latter" with "the <code>Pclass</code> column".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>check.</p>